ratelimit/redis.go

```diff
@@ -147,12 +151,14 @@ func (c *clusterLimitRedis) allow(ctx context.Context, clearText string) (bool,
 	// drop all elements of the set which occurred before one interval ago.
 	_, err := c.ringClient.ZRemRangeByScore(ctx, key, 0.0, float64(clearBefore))
 	if err != nil {
+		c.logError(err)
```

Comment:
I think logError in this method are redundadnt - we log error in the Allow that calls it.